recipes-core/platform-as4630-54pe/platform-as4630-54pe_0.1.bb

```diff
@@ -27,8 +27,8 @@ do_install() {
         install -m 0755 ${WORKDIR}/platform-as4630-54pe-init.sh ${D}${bindir}
 
         # systemd-networkd
-        install -d ${D}${sysconfdir}/systemd/network/
```

Comment:
i assume this directory will still be there in the final image, since systemd-networkd will take care of creating it in another place?

Reply:
Yes, else we would have the issue already on all the other platforms that do not ship a `90-enp.link`.